src/main/main.js

```diff
@@ -1,18 +1,26 @@
 /* global __static */
```

Comment:
This massive list of imports is a pretty good indicator that this file needs some refactoring in the future.

Reply:
yes, a massive one )